### DISCUSSION TITLE
avast Cybercapture

### DISCUSSION THREAD

**Discussion by sellige00** [NickvisionApps/Parabolic#958] [General]
Hi

The new release 2024.11.1  is detected as a problem by Avast Cybercapture

**Comment by nlogozzo** [comment]
Because it's brand new and has never seen it before...

**Comment by nlogozzo** [comment]
What's the "problem" that it says it has?

**Comment by sellige00** [comment]
It prevents the use of parabolic , and it removes the shortcut from the desktop
So I temporarily suppressed the option Cybercapture

With the previous release,  I hadn't any probleme

Anyway thank you for  your software

**Comment by nlogozzo** [comment]
Like I said it's probably because you are the first person with avast to use Parabolic 2024.11.1 and it isn't well known yet.
